agent/agentssh/agentssh.go

```diff
@@ -580,6 +589,14 @@ func (s *Server) Serve(l net.Listener) error {
 }
 
 func (s *Server) handleConn(l net.Listener, c net.Conn) {
+	logger := s.logger.With(
+		slog.F("remote_addr", c.RemoteAddr()),
+		slog.F("local_addr", c.LocalAddr()),
+		slog.F("listen_addr", l.Addr()))
```

Comment:
Considering the Tailscale in-memory addresses, are these valuable output? I recall when was implementing `SSH_CLIENT` and `SSH_CONNECTION` env variables, I was advised against using them:

```go
	// Set SSH connection environment variables (these are also set by OpenSSH
	// and thus expected to be present by SSH clients). Since the agent does
	// networking in-memory, trying to provide accurate values here would be
	// nonsensical. For now, we hard code these values so that they're present.
	srcAddr, srcPort := "0.0.0.0", "0"
	dstAddr, dstPort := "0.0.0.0", "0"
```

Reply:
I want to get TCP flow logs out of gVisor and so having identifiers that we can correlate down into the network stack are very useful for debugging.  They're a bit like pointers: not useful outside the process, but can be highly useful as identifiers in the process context.

Reply:
Makes sense 👍🏻